src/containers/Elements/NodeProviderPanels.js

```diff
@@ -74,30 +74,51 @@ class NodeProviderPanels extends PureComponent {
     }));
   }
 
-  isPanelOpen = index => this.state.panels[index];
+  showPanelAlwaysOpen = (panelIndex, show) => {
+    this.setState(previousState => ({
+      panelsAlwaysOpen: map(
+        previousState.panelsAlwaysOpen,
+        (item, index) => {
+          if (panelIndex === index) { return show; }
+          return item;
+        },
+      ),
+    }));
+  };
+
+  isPanelAlwaysOpen = index => this.state.panelsAlwaysOpen[index];
+
+  anyPanelAlwaysOpen = () => some(
+    map(this.state.panelsAlwaysOpen, (value, index) => this.isPanelAlwaysOpen(index)),
+    show => show === true,
+  );
+
+  isPanelOpen = index => this.state.panelsOpen[index];
 
   anyPanelOpen = () => some(
-    map(this.state.panels, (value, index) => this.isPanelOpen(index)),
+    map(this.state.panelsOpen, (value, index) => this.isPanelOpen(index)),
     open => open === true,
   );
 
   render() {
     const { panels, stage, prompt } = this.props;
 
     return (
-      <Panels minimise={!this.anyPanelOpen()}>
+      <Panels minimise={!this.anyPanelAlwaysOpen() && !this.anyPanelOpen()}>
         { panels.map((panel, panelIndex) => (
           <Panel
             title={panel.title}
             key={panelIndex}
-            minimise={!this.isPanelOpen(panelIndex)}
+            minimise={!this.isPanelAlwaysOpen(panelIndex) && !this.isPanelOpen(panelIndex)}
             highlight={getHighlight(panel, panelIndex)}
           >
             <NodeProvider
               {...panel}
               stage={stage}
               prompt={prompt}
               onUpdateNodes={nodes => this.onUpdatePanelState(panelIndex, nodes.length !== 0)}
+              showPanelAlways={show => this.showPanelAlwaysOpen(panelIndex, show)}
```

Comment:
Node provider probably shouldn't have any knowledge of a panel. Maybe NodeProvider could have something like an `onDrag` prop? and

Reply:
Makes sense.